chart/README.md

```diff
@@ -32,6 +32,10 @@ cluster using the [Helm](https://helm.sh) package manager.
 - Helm 2.11+ or Helm 3.0+
 - PV provisioner support in the underlying infrastructure
 
+## Configuring Airflow
+
+All Airflow configuation parameters (equivalent of `airflow.cfg`) are stored in [values.yaml](https://github.com/apache/airflow/blob/master/chart/values.yaml). You may wish to consult these prior to Chart installation.
+
```

Comment:
The description is not clear enough, it also does not tell which key needs to be set, all configs need to go under `config` key:

https://github.com/apache/airflow/blob/9c75ea3c14b71d2f96d997aeef68c764c7d2984c/chart/values.yaml#L643

Would be better to give an example too

Reply:
Thank you for the suggestion @kaxil I appreciate that. I've augmented the PR to accommodate.